File: elysium-characters-wayward/src/main/java/com/seventh_root/elysium/characters/wayward/race/RaceWrapper.java
package com.seventh_root.elysium.characters.wayward.race;

import net.wayward_realms.waywardlib.character.Race;
import net.wayward_realms.waywardlib.classes.Stat;

import java.util.Map;

public class RaceWrapper implements Race {
    private final com.seventh_root.elysium.api.character.Race elysiumRace;

    public RaceWrapper(com.seventh_root.elysium.api.character.Race elysiumRace) {
        this.elysiumRace = elysiumRace;
    }

    public com.seventh_root.elysium.api.character.Race getElysiumRace() {
        return elysiumRace;
    }

    @Override
    public String getName() {
        return elysiumRace.getName();
    }

    @Override
    public int getStatBonus(Stat stat) {
        return 0;
    }

    @Override
    public Map<String, Object> serialize() {
        return null;
    }

}

File: elysium-characters-wayward/src/main/kotlin/com/seventh_root/elysium/characters/wayward/race/RaceWrapper.kt
package com.seventh_root.elysium.characters.wayward.race

import net.wayward_realms.waywardlib.character.Race
import net.wayward_realms.waywardlib.classes.Stat

class RaceWrapper(val elysiumRace: com.seventh_root.elysium.api.character.Race) : Race {

    override fun getName(): String {
        return elysiumRace.name
    }

    override fun getStatBonus(stat: Stat): Int {
        return 0
    }

    override fun serialize(): Map<String, Any>? {
        return null
    }

}
